README.md

```diff
@@ -1,279 +1,42 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b> Hello Microverse, project</b></h3>
 
 </div>
 
-<!-- TABLE OF CONTENTS -->
-
-# 📗 Table of Contents
-
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
```

Comment:
- [ ] Great job 👍 using the [Microverse template](https://github.com/microverseinc/readme-template) for your README and customizing it to reflect your project as suggested by the previous reviewer.  But I can see 👀 you have removed the content table 🗒️  and missed a lot of sections from the content table. Please make sure you add them into your README as it is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) of the project to use the [template](https://github.com/microverseinc/readme-template) for README and include all sections from the template. The only section that is OPTIONAL to add is FAQ. 
- [ ] Please 🙏  make sure to customize each section to reflect your project
- [ ] Also make sure to remove comments and _**Note**_ after you are done customizing ☺️

Reply:
hello @shella12.

Reply:
I´m sorry but, the content of my first project is very short, just a main title in html and its font color in css.
I only used html and css so I don't have enough material to fill all the sections of the microverse template.
Also, in the 7th line of the microverse template it says this: "Modify this file to match your project and remove sections that don't apply." what can I do?

Reply:
Hey @RuriEnciso I understand that it's difficult 😅  to come up with something like the Future features or key features for such a small project. Microverse requires students to stick to the template as practice for future projects. I believe you can always come up with some content to fill the required section lets say for example Future feature includes Goodbye Microverse or Colors and Fonts lets say. I hope you understand. If you still need further explanation feel free to reach out ☺️